sdcm/sct_events/database.py

```diff
@@ -127,6 +128,8 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                                    regex="Aborting on shard")
 DatabaseLogEvent.add_subevent_type("SEGMENTATION", severity=Severity.ERROR,
                                    regex="segmentation")
+DatabaseLogEvent.add_subevent_type("MALFORMED_SSTABLE", severity=Severity.CRITICAL,
+                                   regex="sstables::malformed_sstable_exception")
```

Comment:
Don't we have more regex suggesting of sstable corruption?
Did you check Scylla code or asked someone?

Reply:
I spoke with Benny, he said that is a good mark. I also found some other indicator: `invalid_mutation_fragment_stream` and I added it to verification.